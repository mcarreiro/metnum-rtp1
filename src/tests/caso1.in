1024 1024 100 64 400 8 
512 100
100 512
900 512
512 900
750 750
250 750
750 250
250 250
