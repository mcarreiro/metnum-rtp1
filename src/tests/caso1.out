0	0	-100.00000
0	1	-100.00000
0	2	-100.00000
0	3	-100.00000
0	4	-100.00000
0	5	-100.00000
0	6	-100.00000
0	7	-100.00000
1	0	-100.00000
1	1	11.39834
1	2	122.79668
1	3	179.78836
1	4	400.00000
1	5	169.49319
1	6	92.37330
1	7	-100.00000
2	0	-100.00000
2	1	122.79668
2	2	400.00000
2	3	296.35678
2	4	316.14618
2	5	285.59949
2	6	400.00000
2	7	-100.00000
3	0	-100.00000
3	1	179.78836
3	2	296.35678
3	3	289.49261
3	4	282.62845
3	5	256.75851
3	6	205.20163
3	7	-100.00000
4	0	-100.00000
4	1	400.00000
4	2	316.14618
4	3	282.62845
4	4	268.11652
4	5	253.60455
4	6	264.04800
4	7	400.00000
5	0	-100.00000
5	1	169.49319
5	2	285.59949
5	3	256.75851
5	4	253.60455
5	5	225.49516
5	6	197.38579
5	7	-100.00000
6	0	-100.00000
6	1	92.37330
6	2	400.00000
6	3	205.20163
6	4	264.04800
6	5	197.38579
6	6	400.00000
6	7	-100.00000
7	0	-100.00000
7	1	-100.00000
7	2	-100.00000
7	3	-100.00000
7	4	400.00000
7	5	-100.00000
7	6	-100.00000
7	7	-100.00000
